bin/run.sh

```diff
@@ -33,14 +33,14 @@ case $1 in
     if [[ $1 == "backend" ]]; then
         shift
         run_back_end_tests
+        coverage report -m
```

Comment:
Thanks for the pointers. I'll refer back to this. I was looking at atmo as an example and saw it would require a bit more effort so I went with just the local report for now. It's above 90% so that's good. 👍